manifests/init.pp

```diff
@@ -80,11 +88,13 @@
             onlyif   => "! grep '^exclude=' /etc/yum.conf",
             user     => root,
             provider => shell,
-            require  => Exec['Ensure Service']
+            require  => Exec['Ensure Service'],
         }->
         exec { 'Yum Exclude gitlab-ci-multi-runner':
-            command  => "sed -i 's/^exclude=.*$/& gitlab-ci-multi-runner/' /etc/yum.conf",
-            onlyif   => "! grep '^exclude=.*gitlab-ci-multi-runner' /etc/yum.conf",
+            command  => "sed -i 's/^exclude=.*$/& gitlab-ci-multi-runner/' \
```

Comment:
When working from terminal with multiple files/split it could be nice to have short lines, but yes I tend to agree with you that 80 is sometime realy too short. If you want I can use the recommeded two space indentation (instead of the four spaces used in this project). I will put back the check inline.